eng/source-build-patches/0012-Don-t-warn-on-CA1416.patch

```diff
@@ -1,26 +0,0 @@
-From 446691a345e68861bd653c1be4b9111aafad5613 Mon Sep 17 00:00:00 2001
```

Comment:
TODO: what failures are we seeing that required this patch?

Reply:
Hmm I didn't see any warnings/errors after removing this patch, I guess we just don't need this then? Can you double check why this was added originally @MichaelSimons ?

Reply:
Feel free to remove it.  This could be a legacy patch that is no longer needed.  We would prefer to clean up the patches than blindly bring them forward.